src/app/components/navbar/navbar.component.scss

```diff
@@ -174,4 +174,37 @@ ul.top-menu > li > a:hover .caret {
   li.nav-item:last-child {
     margin-top: 0.5rem;
   }
-}
\ No newline at end of file
+}
+.dropbtn {
+  font-size: 16px;
+  border: none;
+}
+
+.dropdown {
+  position: relative;
+  display: inline-block;
+}
+
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: #f1f1f1;
+  min-width: 160px;
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+  z-index: 1;
+}
+
+.dropdown-content a {
```

Comment:
@patrickf949 since we are using SCSS, I recommend you consider wrapping styles of children elements under there parent class name. i.e for `dropdown-content` you define the children classes  like below
```
.dropdown-content {
  a {
       color: black;
       &: hover {

       }
    }
}
```